polymesh_schema.json

```diff
@@ -941,9 +941,11 @@
             "valid_from": "Option<Moment>"
         },
         "MercatTxData": {
-            "InitializedTransfer": "Text",
-            "FinalizedTransfer": "Text",
-            "JustifiedTransfer": "Text"
+            "_enum": {
+                "InitializedTransfer": "Text",
+                "FinalizedTransfer": "Text",
+                "JustifiedTransfer": "Text"
```

Comment:
These may need to be `Vec<u8>`. 

`Text` implies ASCII text while `Vec<u8>` implies hex/binary data.

Reply:
These aren't my changes, it's from a commit that was brought over when I rebased on latest the `mercat`.